frontend/src/routes/+page.svelte

```diff
@@ -11,7 +26,11 @@
   <TgParagraph>
     This is an app for "pointing" agile stories. To get started
   </TgParagraph>
-  <TgButton id="createRoomButton" type="primary" href="/rooms" class="mt-4"
-    >Create a room</TgButton
-  >
+  {#if status}
+    <TgButton id="createRoomButton" type="primary" href="/rooms" class="mt-4"
+      >Create a room</TgButton
+    >
+  {:else}
+    <div id="apiCheck" class="text-red-500 font-bold">API IS UNAVAILABLE</div>
```

Comment:
Just a couple changes here:

- Can you change the message to be "The Guesstimator is currently unavailable, probably due to higher than expected demand. Please try again later, or if the problem persists for more than a day you can [submit a bug report](https://github.com/skill-collectors/guesstimator/issues/new?assignees=&labels=bug&template=bug_report.md&title=)." I've been trying to generally follow [this advice](https://wix-ux.com/when-life-gives-you-lemons-write-better-error-messages-46c5223e1a2f) for error messages.
  - The `<div>` should be a `<TgParagraph>` and the link should be a `<TgLink>`.
- Please use `orange-800` for the color, since that matches the "danger" button on the style demo and the "delete room" button.

Reply:
Thanks, I'll make the changes... it seems my linter really wants to break those lines up for some reason, so that'll take some tweaking.

Reply:
Is this block more along the lines of what you're looking for?


`
<TgParagraph class="apiCheck text-orange-800 font-bold">
      The Guesstimator is currently unavailable, probably due to higher than
      expected demand. Please try again later, or if the problem persists for
      more than a day you can
      <TgLink href="https://github.com/skill-collectors/guesstimator/issues/
        new?assignees=&labels=bug&template=bug_report.md&title=">
        submit a bug report.
      </TgLink>
    </TgParagraph>
`

Reply:
Yes, that's pretty close. I actually don't think you need `apiCheck` to be a class or id, though.

Regarding the formatting, the project is set up to use Prettier and ESLint, and the build in GitHub Actions will fail if the code doesn't match the expected style. So you should generally just let them do whatever they want with line breaks etc.. If you use VSCode, it should use [the project's settings](https://github.com/skill-collectors/guesstimator/blob/main/.vscode/project.code-workspace#L27) to auto-format when you save.